data-processing/tests/test_parse_tex.py

```diff
@@ -447,6 +442,16 @@ def test_extract_bibitem_stop_at_newline():
     assert bibitems[0].text == "token1"
 
 
+def test_extract_bibitem_include_hyperref_contents():
```

Comment:
actually, do you have an example arXiv ID you can paste in comment here? want to see a real example of this

Reply:
Sure thing! Take for example paper 1702.01287 (https://arxiv.org/pdf/1702.01287.pdf).

In the references section, many of the references have the titles as hyperlinks, such as this one:

![image](https://user-images.githubusercontent.com/2358524/103700428-46984500-4f59-11eb-8ee6-e9acd67382b7.png)

The BBL file for this looks like:

```latex
\bibitem[{Bengio et~al.(2003)Bengio, Ducharme, Vincent, and
  Janvin}]{Bengioetal2003}
Yoshua Bengio, R{\'e}jean Ducharme, Pascal Vincent, and Christian Janvin. 2003.
\newblock \href{http://dl.acm.org/citation.cfm?id=944919.944966}{{A Neural
  Probabilistic Language Model}}.
\newblock {\em J. Mach. Learn. Res.\/} 3:1137--1155.
\newblock
  \href{http://dl.acm.org/citation.cfm?id=944919.944966}{http://dl.acm.org/citation.cfm?id=944919.944966}.
```

where the title apears in an `\href` command: `\href{http://dl.acm.org/citation.cfm?id=944919.944966}{{A Neural
  Probabilistic Language Model}}`. For some reason, the text for those commands was not getting included in the text we extracted from the BBL files.

In case this example would be useful for future reference, I've added it as a comment to the main thread for this PR.